src/graphql/query/portfolio/myLoans.graphql

```diff
@@ -0,0 +1,61 @@
+query myLoans {
+	my {
+		id
+		loans {
```

Comment:
It looks like `my > loans` has a default limit of `20`, so this query is probably returning only `20` loans. I'm guessing we want a way to get all loans?

Reply:
It's built into the query, so set in the backend, and there's an `offset` parameter. You can see the fields in the sandbox for dev or prod: https://gateway.development.kiva.org/graphql

Reply:
It would be worth testing with an account to see if the limit is indeed respected. But all of our queries that I've seen with `values` returned are setup for paging with `limit` and `offset`.

Reply:
I am currently pulling 20 from my account so it does seem like the limit is being respected.  I can add pagination in a future commit if that works...

Reply:
Yeah we'll need paging to make this a replacement for the old feature. No worries if it's a following ticket.